### PR TITLE
Added exit code to run-with-logging.sh

### PR DESCRIPTION
**Pull request by tfokken**

### CODE REVIEW

File: templates/shared/run-with-logging.sh [hunk 1]
```diff
@@ -43,6 +43,7 @@ table_name=`basename "$PWD"`
 if [ $check_b -ne 0 ]
         then
                 echo -e "$timestamp|$table_name|${RED}FAIL${NC}: $@ " 2>&1 | tee -a  $STATUS_LOG
+		exit 1;
```
Comment (afoerster) [inline templates/shared/run-with-logging.sh]: Can it exit `$check_b` instead?

Comment (tfokken) [reply templates/shared/run-with-logging.sh]: yes

**Comment by davececchi** [comment]
I didn't dig into this but merging code with a failing build needs an explanation.

**Comment by afoerster** [comment]
@davececchi  The change didn't break the failing tests, it just exposed them. 

I wanted to get that fix in here ASAP so we aren't masking failures. We are working on some fixes for the failing tests, but exiting with the right failure code and having a correct application was the priority.

**Comment by davececchi** [comment]
Thanks for explanation.  Seems unlike you to break the glass on something that only needed another 40 minutes of development to be right, but presumably your judgement remains sound ;-)

**Pull request merged**